### ISSUE TITLE
Parsing of datetime produces debug log as warning

### ISSUE THREAD

**Issue by PascalTurbo** [feedjira/feedjira#395]
When parsing a datetime that isn't supported and logger is set to log level "WARN" the debug output aka backtrace is logged with severity WARN instead of DEBUG.

**Comment by mikeastock** [comment]
This has been released in https://github.com/feedjira/feedjira/releases/tag/v2.1.3

**Comment by mockdeep** [comment]
@Ben-Fenner can you give version 3.2.3 a try?

**Comment by Ben-Fenner** [comment]
Sorry, version `3.1.1` is the latest I can manage right now, but I did confirm the code causing the regression was still in place when I made my post. Sorry, I should have linked to it.

I went to go find it to give you a link to the code just now, but it looks like it's been fixed. And looking at the file history, it seems you did so about 3 days ago. https://github.com/feedjira/feedjira/commit/336b6581cf6a0968af840b33397cf03677ffe045

I see now you're hoping I can test that out. Sorry I can't test because I'm on Ruby `2.3.8` but for what it's worth it looks like your commit should do the trick. Thank you for the work. I wistfully hope one day soonish we'll be on a newer Ruby version.

For those who find this issue in the future and need a work-around like I did. We added a Rails initializer that changes the log level to ERROR.

_config/initializers/feedjira.rb_
```RUBY
# Feedjira's logging for datetime parse issues is set to WARN, and so is the resulting backtrace output, when both (or
# at least the backtrace) should be set to DEBUG. This was a problem in `2.1.2` and was fixed in `2.1.3` but returned as
# a regression some time later and the version we're on now (`3.1.1`) is misbehaving again. So we set the log level here
# abnormally high to avoid that situation.
#
# Refs:
#  https://github.com/feedjira/feedjira/issues/397
#  https://github.com/feedjira/feedjira/issues/395
#  https://github.com/feedjira/feedjira/blob/v3.1.1/lib/feedjira/configuration.rb
#  https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
#
# TODO: It looks like version 3.2.3 has the fix for this, so upgrade when possible.
Feedjira.configure do |config|
  config.logger.level = Logger::ERROR
end
```
